CONTRIBUTING.md

```diff
@@ -14,22 +14,22 @@ Before submitting the code, we need to do some preparation:
 
 1. Sign up or login to GitHub:  [https://github.com](https://github.com)
 
-2. Fork HugeGraph repo from GitHub: [https://github.com/hugegraph/hugegraph/fork](https://github.com/hugegraph/hugegraph/fork)
+2. Fork HugeGraph repo from GitHub: [https://github.com/apache/incubator-hugegraph/fork](https://github.com/apache/incubator-hugegraph/fork)
 
-3. Clone code from fork repo to local: [https://github.com/${GITHUB_USER_NAME}/hugegraph](https://github.com/${GITHUB_USER_NAME}/hugegraph)
+3. Clone code from fork repo to local: [https://github.com/${GITHUB_USER_NAME}/incubator-hugegraph](https://github.com/${GITHUB_USER_NAME}/incubator-hugegraph)
 
    ```shell
    # clone code from remote to local repo
-   git clone https://github.com/${GITHUB_USER_NAME}/hugegraph
+   git clone https://github.com/${GITHUB_USER_NAME}/incubator-hugegraph.git
```

Comment:
prefer to keep "hugegraph" directory name: `git clone git-url directory-name`

Reply:
Done.
THANKS